.rake_tasks~

```diff
@@ -0,0 +1,48 @@
+about
```

Comment:
Moreover, it seems you .gitignore failed.

Reply:
I wonder why, I'll test this out now.